### PR TITLE
@alloy -> fix memory leak that casued failing tests

### PR DESCRIPTION
**Pull request by 1aurabrown**
Going to have a few small PRs coming in today. Here's the one that fixed the web view memory leak issue.

### CODE REVIEW

File: Artsy/Classes/View Controllers/ARExternalWebBrowserViewController.m [hunk 1]
```diff
@@ -99,4 +99,12 @@ - (NSDictionary *)dictionaryForAnalytics
     return nil;
 }
 
+- (void)dealloc
+{
+    [self.webView loadHTMLString:@"" baseURL:nil];
+    [self.webView stopLoading];
+    self.webView.delegate = nil;
+    [self.webView removeFromSuperview];
```
Comment (alloy) [inline Artsy/Classes/View Controllers/ARExternalWebBrowserViewController.m]: Can you add a comment summarising the issue and solution and a link to the best write-up on it?

**Comment by alloy** [comment]
Well done on this, Sherlock! :clap:

File: Artsy/Classes/View Controllers/ARExternalWebBrowserViewController.m (hunk 1 above)
Comment (1aurabrown) [inline Artsy/Classes/View Controllers/ARExternalWebBrowserViewController.m]: done

Comment (alloy) [inline Artsy/Classes/View Controllers/ARExternalWebBrowserViewController.m]: 👌🏿

**Comment by 1aurabrown** [comment]
cocoapods travis prob

**Pull request merged**